packages/workers-shared/router-worker/src/worker.ts

```diff
@@ -75,6 +77,69 @@ export default {
 
 			const maybeSecondRequest = request.clone();
 
+			if (config.static_routing) {
```

Comment:
Do we also need to check `config.has_user_worker` here or is `config.static_routing` only included if there is a User worker?

Reply:
static_routing can only be provided when there is a user Worker, yes. if for some reason there are static_routing rules provided without a user Worker, and a rule resolves to the user Worker, we'll throw before attempting to dispatch there